MapboxCoreNavigation/RouteController.swift

```diff
@@ -165,6 +228,78 @@ open class RouteController: NSObject {
         locationManager.stopUpdatingLocation()
         locationManager.stopUpdatingHeading()
     }
+    
+    /**
+     Send feedback about the current road segment/maneuver to the Mapbox data team.
+     
+     You can pair this with a custom feedback UI in your app to flag problems during navigation
+     such as road closures, incorrect instructions, etc. 
+     
+     With the help of a custom `description` to elaborate on the nature of the problem, using
+     this function will automatically flag the road segment/maneuver the user is currently on for 
+     closer inspection by Mapbox's system and team.
+     
+     If you provide a custom feedback UI that lets users elaborate on an 
+     issue, you should call this before you show the custom UI so the 
+     location and timestamp are more accurate. You can then call 
+     `updateLastFeedback()` to attach any additional metadata to the 
+     feedback.
+     */
+    public func recordFeedback(type: FeedbackType, description: String?) {
+        enqueueFeedbackEvent(type: type, description: description)
+    }
+    
+    /**
+     Update the last recorded feedback event, for example if you have a custom feedback UI that lets a user elaborate on an issue.
+     */
+    public func updateLastFeedback(type: FeedbackType, description: String?) {
+        if let lastFeedback = outstandingFeedbackEvents.filter({$0 is FeedbackEvent}).last {
+            lastFeedback.eventDictionary["feedbackType"] = type.rawValue
```

Comment:
`FeedbackType.rawValue` is an integer (e.g., 0 for `general`). Did you mean to set this item to `type.description` instead?